recipes/sunpy/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
     - matplotlib
     - pandas
     - mock
-    - nomkl  # [win]
+    - mkl =11.3.1  # [win]
```

Comment:
try this:
```yaml
- mkl 11.3.1  # [win]
```
example here with libtiff (https://github.com/conda-forge/pillow-feedstock/blob/6c0fb50ddbd04a9a3e2302723c840c628494ed01/recipe/meta.yaml)